src/commcare_cloud/ansible/roles/kernel_tune/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- include: thp.yml
+  tags: thp
+- include: disk_scheduler.yml
+  tags: disk_scheduler
```

Comment:
Nit: newline at the end of the file (maybe configure your editor to do this for you: [pycharm example](https://stackoverflow.com/questions/36043061/automatically-add-newline-on-save-in-pycharm))